packages/nimble-components/src/text-area/template.ts

```diff
@@ -0,0 +1,70 @@
+import { html, ref, slotted, ViewTemplate } from '@microsoft/fast-element';
+import type { FoundationElementTemplate } from '@microsoft/fast-foundation';
+import type { TextArea } from '.';
+import { iconExclamationMarkTag } from '../icons/exclamation-mark';
+import { errorTextTemplate } from '../patterns/error/template';
+
+export const template: FoundationElementTemplate<
+ViewTemplate<TextArea>
+> = () => html<TextArea>`
+    <template
+        class="${x => (x.hasVerticalScrollbar ? 'vert-scrollbar' : '')}">
```

Comment:
I've moved away from toggling a class at all, so this is now moot.

Reply:
Re-opening as the newly added `--ni-private-scrollbar-width` has the same concern, except in this case mutating the public style attribute instead of the class attribute when it seems possible to avoid.

Reply:
Now we're setting the variable by template binding on the icon.